README.md

```diff
@@ -44,6 +44,12 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Garvin Chimone Personal Portfoilo](https://chimoneg27.github.io/Personal-Portfolio/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 ## 💻 Getting Started <a name="getting-started"></a>
 
```

Comment:
- [ ] The issue in your code is with the incorrect referencing of the relative path to the image in your `desktop.css` file. On line 206, you have the following code:

```css
background-image: url(/images/desktop-images/Img\ Placeholdertwo.png);
```

- The problem lies in two aspects. First, the path to the image is not correctly specified using relative paths. It should be `url(../images/desktop-images/ImgPlaceholdertwo.png);`. The `..` indicates going up one level from the current CSS file's location, then entering the `images` folder and further navigating to the `desktop-images` folder.

- Secondly, there are spaces within the image file name, which is not recommended. It's better to remove the spaces and use a file name like `ImgPlaceholdertwo.png`. Rename all your images removing all spaces.

To fix the issue, update the code to the following:

```css
background-image: url(../images/desktop-images/ImgPlaceholdertwo.png);
```

These changes ensure that the relative path is correctly referenced and that the image file name is free of spaces, resolving the issue with displaying the image.

<img width="443" alt="image" src="https://github.com/Chimoneg27/Personal-Portfolio/assets/84156714/1c6df093-ac18-4bf1-b0ee-c710ce12a9c8">

Reply:
Thanks